### DISCUSSION TITLE
MadMax plotter keeps asking for -p value

### DISCUSSION THREAD

**Discussion by quint641** [madMAx43v3r/chia-plotter#763] [Q&A]
Hi there,

I might misread something because of the language barrier, but you're able to plot with the -c value right? (you need to put in your pool address).

But when I use: `./build/chia_plot -r 31 -t /chia/plot/ -2 /chia/ramdisk/ -d /chia/farm/hdd01/ -c PoolAddress'

It keeps asking for: `Pool Public Key (48 bytes) needs to be specified via -p <hex>, see chia keys show.`

Since i'm plotting and farming on the same machine I dont need -f if I'm correct.
Don't know what I'm doing wrong, might hope that someone knows what I oversee. I'm running on Ubuntu 20.4 btw.

**Comment by gryan315** [comment]
Are you using main branch or pool puzzles branch?

**Reply by quint641** [reply]
@gryan315 Just noticed I need the puzzles branch yeah. Quite new to git. How does one het the puzzle branch?

**Reply by gryan315** [reply]
`git checkout branchname`
`git pull origin branchname`
In this case, the name of the branch is `pool-puzzles`

**Reply by SebMoore** [reply]
> `git checkout branchname`
> `git pull origin branchname`
> In this case, the name of the branch is `pool-puzzles`

Please note - as of about 5 minutes ago, the pool-puzzles branch has been merged into master. Hence, you no longer need the pool puzzles branch. Just re-download the master branch or use
`git checkout master`
`git pull`
`git submodule update --init`
to update to the latest version and enable pool plotting.

**Reply by gryan315** [reply]
>as of about 5 minutes ago, the pool-puzzles branch has been merged into master.

I've been refreshing the page waiting for the merge, must have missed it while I was explaining how to get a branch.😆

**Reply by quint641** [reply]
@gryan315 @VertiHydro Thanks for the help guys!

**Reply by SebMoore** [reply]
> > as of about 5 minutes ago, the pool-puzzles branch has been merged into master.
> 
> I've been refreshing the page waiting for the merge, must have missed it while I was explaining how to get a branch.😆

a watched pot never boils... 😂

**Reply by AjazElahi** [reply]
I'm still having this problem. I've tried updating and then starting the plotter again but I keep getting the same warning because I'm not using -p any ideas please? My code is: 
`./build/chia_plot -n 1 -r 32 -u 256 -v 256 -c <PoolContractAddress> -f <Farmer Address> -t /media/blade3/plotter/ -2 /mnt/ram/ -d /media/blade3/farmdsk1/Portable/`

Keep seeing error: 
`Pool Public Key (48 bytes) needs to be specified via -p <hex>, see chia keys show.`

**Reply by SebMoore** [reply]
Hmmm. That is odd. Bit of a long shot, but try putting your -t and -2 and -d before your -c and -f : that's what works for me but I don't know why it would be any different

**Reply by gryan315** [reply]
Did you rebuild with `./make_devel.sh`?

**Reply by AjazElahi** [reply]
Just to update, I forgot to rebuilt with `./make_devel.sh` did that and it worked fine :)

**Reply by chalamb1** [reply]
Im having this same issue. Using windows. Madmax thru powershell.  What do i do to fix it? Please. Im a noob so be specific

**Reply by gryan315** [reply]
Windows is the stotiks build, download the new 0.1.1 or whatever version it is.
